x-pack/plugins/aiops/public/shared_lazy_components.tsx

```diff
@@ -25,6 +27,6 @@ const LazyWrapper: FC = ({ children }) => (
  */
 export const ExplainLogRateSpikes: FC<ExplainLogRateSpikesProps> = (props) => (
   <LazyWrapper>
-    <ExplainLogRateSpikesLazy {...props} />
+    <ExplainLogRateSpikesWrapperLazy {...props} />{' '}
```

Comment:
Guess the {' '} part can be removed, might be a leftover from an auto-format gone wrong.